index.js

```diff
@@ -153,7 +156,8 @@ module.exports = function(cache, defaults){
             }
             if(!err && cachedResponse && policy
               && policy.satisfiesWithoutRevalidation(Request.toJSON()) && !props.forceUpdate) {
-              return utils.callbackExecutor(cb, null, cachedResponse, key, Request);
+              // Return the clone of the cached response.
```

Comment:
The reason for cloning is that the response kept & returned from cache is to be modified (eg. setting the x-request-id - this could lead to some inconsistence in responses of subsequent calls to the same endpoint.
I came across this situation in some test, when keeping 2 responses as local variables and trying to assert them later.